packages/react-native-reanimated/android/build.gradle

```
@@ -445,20 +446,31 @@ def assertMinimalReactNativeVersion = task assertMinimalReactNativeVersionTask {
     }
 }
 
-task prepareHeadersForPrefab(type: Copy) {
-    from("$projectDir/src/main/cpp/reanimated")
+task prepareWorkletsHeadersForPrefabs(type: Copy) {
+    from("$projectDir/src/worklets/cpp")
+    from("$projectDir/../Common/cpp/worklets/NativeModules")
+    from("$projectDir/../Common/cpp/worklets/Registries")
+    from("$projectDir/../Common/cpp/worklets/SharedItems")
+    from("$projectDir/../Common/cpp/worklets/Tools")
+    from("$projectDir/../Common/cpp/worklets/WorkletRuntime")
```

I think we should be able to get rid of that (actually, most likely we should do it in one of my previous PRs)
```suggestion
    from("$projectDir/../Common/cpp/worklets")
```